### DISCUSSION TITLE
AttributeError: 'NoneType' object has no attribute 'configure'

### DISCUSSION THREAD

**Discussion by divs17** [hacksider/Deep-Live-Cam#662] [Q&A]
Getting this issue everythime I run the run.py. Please help me :(

**Comment by ikattz** [comment]
# For Mac (с Homebrew):
brew install ffmpeg

# For Linux (Ubuntu/Debian):
sudo apt install ffmpeg

# For Windows (с Chocolatey):
choco install ffmpeg

This helped me on M1 Apple Silicon
